enclave/sgx/collateralinfo.c

```diff
@@ -141,8 +246,8 @@ void oe_prealloc_quote_verification_collateral_args(
  * @param[in] default_sizes The default quote verification collateral sizes.
  */
 void oe_update_default_collateral_arg_sizes(
-    oe_get_sgx_quote_verification_collateral_args_t* src_args,
-    oe_get_sgx_quote_verification_collateral_args_t* default_sizes);
+    oe_get_sgx_quote_verification_collateral_with_params_args_t* src_args,
```

Comment:
Not new in this PR so ok to leave if you want, but "src" violates [OE coding style](https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md).